### PR TITLE
WiP docs(v2): Docusaurus 2 docs

### PR DESCRIPTION
**Pull request by wgao19**
## Motivation

Opening this PR for review and discussion on work in progress docs. I will brief my thoughts in follow up comments. And I'll PR complete pieces incrementally.

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes.

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)

NA

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

File: website/docs/configuration.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Docusaurus has a unique take on configurations. We encourage you to congregate i
 
 Keeping a well-maintained `docusaurus.config.js` helps you, your collaborators, and your open source contributors be able to focus on docs while having certain fields easy to customize.
 
-For reference to each of the configurable fields, you may refer to the API reference of [docusaurus.config.js](/docs/docusaurus.config.js).
+For reference to each of the configurable fields, you may refer to the API reference of [docusaurus.config.js](docusaurus.config.js).
```
Comment (endiliey) [inline website/docs/configuration.md]: .md

File: website/docs/configuration.md [hunk 2]
```diff
@@ -24,16 +24,14 @@ The configurations can be categorized into:
 - [Theme configurations, plugins, and presets](#theme-plugins-and-presets-configurations)
 - [Custom configurations](#custom-configurations)
 
+For more details on the API, check out [the API Reference on `docusaurus.config.js`](docusaurus.config.js)
```
Comment (endiliey) [inline website/docs/configuration.md]: .md

File: website/docs/introduction.md [hunk 3]
```diff
@@ -10,61 +10,81 @@ This is a **very early and incomplete alpha release** of Docusaurus 2. We are ma
...
+- :x: You need a full production-ready solution (<small>try [Docusaurus 1](https://docusaurus.io/) instead</small>)
+- :x: You prefer not to work with potential breaking changes and/or features not yet working properly as we improve it during alpha period
 
-## Introduction
+## The next Docusaurus is coming to town
 
 <img src="https://docusaurus.io/img/slash-introducing.svg" alt="Docusaurus"/>
 
-Docusaurus 1 used to be a documentation site generator. In Docusaurus 2, we rebuilt it from the ground up, allowing for more customizability but preserved the best parts of Docusaurus 1 - easy to get started. Docusaurus 2 is a **performant static site generator** can be used to create most content-driven websites (e.g. Documentation, Blogs, Product Landing and Marketing Pages, etc) extremely fast. Out of the box, we have support for documentation-style and blogging websites.
+Docusaurus 1 used to be a pure documentation site generator. In Docusaurus 2, we rebuilt it from the ground up, allowing for more customizability but preserved the best parts of Docusaurus 1 - easy to get started, versioned docs, and i18n.
```
Comment (endiliey) [inline website/docs/introduction.md]: lets omit this part first, just “easy to get started”. versioning and i18n is not yet done

File: website/docs/introduction.md [hunk 4]
```diff
@@ -10,61 +10,81 @@ This is a **very early and incomplete alpha release** of Docusaurus 2. We are ma
...
-## Features
+While our main focus will still be helping you get your documentations right and well, the shared nature of building this static site generator makes it possible for you to build your whole website interface one-stop.
 
-- ⚛️ **Built using React** - Extend or customize with React
-- 📦 **Uses webpack** - Build an optimized bundle and use any CSS-in-JS solution you like
-- ⚡️ **Lightning Fast** - Link prefetching, instant navigation and page views
-- ✂️ **Easy** - Hot reloading out-of-the-box, automatic route-based code and data splitting! Create new routes by creating new Markdown or JSX files
-- 💥 **Single-page Application** - Client-side navigations and easily add dynamic interactivity to your website (Client-side Rendering)
-- 🎯 **SEO Friendly** - `index.html` files are statically generated for every possible path (Static Rendering)
-- 😌 **Painless** - Painless project setup. First class support for documentation
```
Comment (endiliey) [inline website/docs/introduction.md]: why is the comparison removed ? also i think the feature as bullet points are more appealing too.

File: website/docs/introduction.md (hunk 3 above)
Comment (endiliey) [reply website/docs/introduction.md]: also i like the previous text of can be created to build blog, docs out of the box

File: website/docs/introduction.md (hunk 4 above)
Comment (yangshun) [reply website/docs/introduction.md]: Yeah I think the comparison should stay. People won't need to spend the effort to figure out what the differences are.

I'm neutral about the format of features.

Comment (endiliey) [reply website/docs/introduction.md]: with the comparison docs, the docs will be too long. i hope the introduction docs itself is not too long.

Comment (wgao19) [reply website/docs/introduction.md]: OK I will bring back the comparisons. I am not against it just that I prefer not to compare in a way that says how others don't fit. So I'll rephrase part of those.

Comment (endiliey) [reply website/docs/introduction.md]: yup at least keep the comparisons.

Comment (yangshun) [reply website/docs/introduction.md]: > I prefer not to compare in a way that says how others don't fit.

Our intention is to state facts and what each tool is good for. The intention is not to criticize. As a user I'll appreciate fair and accurate comparisons that saves me time from doing it myself.

> I'll rephrase part of those

Yes please 😄

**Comment by wgao19** [comment]
Document #1581
2019年6月7日 23:59 +0800、Yangshun Tay <notifications@github.com>のメール:

**Comment by wgao19** [comment]
Closing this in favor of PRs for individual pieces.

**Pull request closed without merging**